### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by michaelmior** [cmu-db/cmdbac#21]
Is this project still active in any way? I see the last update to any projects happened almost two years ago.

**Comment by apavlo** [comment]
Not really. The website is still running but we haven't run the crawler in a while. Is there something that you need in particular?

P.S.: I am RIT alum.

**Comment by michaelmior** [comment]
@apavlo That's a new fact for me :) I was just interested in seeing data continue to be updated as I'm thinking of using some of it for a project. Wondering if I should think about running my own instance.

**Comment by apavlo** [comment]
@michaelmior I think that we could try to run the crawler again. Send me a separate email to coordinate.
